From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Antonios Christidis <a-christidis@ti.com>
Date: Tue, 24 Feb 2026 12:09:16 -0600
Subject: [PATCH] FROMLIST: dt-bindings: gpu: img: Add J721E compatible

Add the J721E SoC-specific compatible and constrain its single clock and
the GE8430 GPU's two power domains.

Link: https://lore.kernel.org/all/20260224-gpu_dts-v1-2-cc5ddffe140c@ti.com/
Signed-off-by: Antonios Christidis <a-christidis@ti.com>
[Andrei Aldea: order the binding before its user, preserve the separating
 blank line, and add the reviewed GE8430 power-domain constraints.]
Signed-off-by: Andrei Aldea <andrei1998@gmail.com>
---
 .../devicetree/bindings/gpu/img,powervr-rogue.yaml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml b/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml
index a1f54dbae3f3..226de1fdbf9c 100644
--- a/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml
+++ b/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml
@@ -44,6 +44,11 @@ properties:
               - ti,j721s2-gpu
           - const: img,img-bxs-4-64
           - const: img,img-rogue
+      - items:
+          - enum:
+              - ti,j721e-gpu
+          - const: img,img-ge8430
+          - const: img,img-rogue
 
       # This legacy combination of compatible strings was introduced early on
       # before the more specific GPU identifiers were used.
@@ -103,6 +108,7 @@ allOf:
               - ti,am62-gpu
               - ti,am62p-gpu
               - ti,j721s2-gpu
+              - ti,j721e-gpu
     then:
       properties:
         clocks:
@@ -144,6 +150,7 @@ allOf:
           contains:
             enum:
               - img,img-bxs-4-64
+              - img,img-ge8430
               - img,img-ge7800
               - img,img-gx6250
     then:
-- 
2.53.0
